docs/usage/security/excluding-and-including-endpoints.rst

```diff
@@ -1,16 +1,26 @@
 Excluding and including endpoints
 =================================
 
-Please make sure you read the :doc:`security backends documentation </usage/security/security-backends>` first for learning how to set up a security backend. This section focuses on configuring the ``exclude`` rule for those backends.
+Please make sure you read the :doc:`security backends documentation </usage/security/security-backends>` first for
+learning how to set up a security backend. This section focuses on configuring the ``exclude`` rule for those backends.
 
-There are multiple ways for including or excluding endpoints in the authentication flow. The default rules are configured in the ``Auth`` object used (subclass of :class:`~.security.base.AbstractSecurityConfig`). The examples below use :class:`~.security.session_auth.auth.SessionAuth` but it is the same for :class:`~.security.jwt.auth.JWTAuth` and :class:`~.security.jwt.auth.JWTCookieAuth`.
+There are multiple ways for including or excluding endpoints in the authentication flow. The default rules are
```

Comment:
is the new newlines addition intended here and just the next change ?

Reply:
I added line breaks here because the lines were so long they didn't fit on my screen :)

GH is also displaying this diff a bit weird. If you look [at the raw file](https://raw.githubusercontent.com/litestar-org/litestar/5d8dbbcace031c29c237de379899e657f15f1e1e/docs/usage/security/excluding-and-including-endpoints.rst), it doesn't look that odd!

Reply:
ok so that's a welcome addition :)